operations/commit_queue.go

```diff
@@ -274,6 +278,87 @@ func enqueuePatch() cli.Command {
 	}
 }
 
+func backport() cli.Command {
+	return cli.Command{
+		Name:  "backport",
+		Usage: "automatically backport low-risk commits",
+		Flags: mergeFlagSlices(
+			addPatchFinalizeFlag(),
+			addVariantsFlag(),
+			addTasksFlag(),
+			addProjectFlag(),
+			addPatchAliasFlag(),
+			addPatchBrowseFlag(
+				cli.StringFlag{
+					Name:  joinFlagNames(existingPatchFlag, "e"),
+					Usage: "existing commit queue patch",
+				},
+			)),
+		Before: mergeBeforeFuncs(
+			setPlainLogger,
+			requireProjectFlag,
```

Comment:
Does this project denote the project to backport to? In which case maybe you should add a new flag instead of addProjectFlag that gives a more relevant usage message. If it denotes the project that the existing patch is part of, then I don't think it's necessary since the existing patch is given.

Reply:
>Does this project denote the project to backport to?

yes. I've replaced it with a `-b, --backport-project` flag